record_aaaa.go

```diff
@@ -0,0 +1,127 @@
+package ibclient
+
+import (
+	"fmt"
+)
+
+type RecordAAAAOperations interface {
+	CreateAAAARecord(recA4 RecordAAAA) (*RecordAAAA, error)
+	GetAAAARecord(recA4 RecordAAAA) (*[]RecordAAAA, error)
+	DeleteAAAARecord(recA4 RecordAAAA) (string, error)
+	UpdateAAAARecord(recA4 RecordAAAA) (*RecordAAAA, error)
+}
+
+type RecordAAAA struct {
+	IBBase            `json:"-"`
+	Ref               string `json:"_ref,omitempty"`
+	Ipv6Addr          string `json:"ipv6addr,omitempty"`
+	Name              string `json:"name,omitempty"`
+	View              string `json:"view,omitempty"`
+	Zone              string `json:"zone,omitempty"`
+	Ea                EA     `json:"extattrs,omitempty"`
+	NetView           string `json:"omitempty"`
+	Cidr              string `json:"omitempty"`
+	AddEA             EA     `json:"omitempty"`
+	RemoveEA          EA     `json:"omitempty"`
+	CreationTime      int    `json:"creation_time,omitempty"`
+	Comment           string `json:"comment,omitempty"`
+	Creator           string `json:"creator,omitempty"`
+	DdnsProtected     bool   `json:"ddns_protected,omitempty"`
+	DnsName           string `json:"dns_name,omitempty"`
+	ForbidReclamation bool   `json:"forbid_reclamation,omitempty"`
+	Reclaimable       bool   `json:"reclaimable,omitempty"`
+	Ttl               uint   `json:"ttl,omitempty"`
+	UseTtl            bool   `json:"use_ttl,omitempty"`
+}
+
+// NewRecordAAAA creates a new AAAA Record type with objectType and returnFields
+func NewRecordAAAA(ra RecordAAAA) *RecordAAAA {
+	res := ra
+	res.objectType = "record:aaaa"
+	res.returnFields = []string{"ipv6addr", "name", "view", "zone", "extattrs", "comment", "creation_time",
+		"creator", "ddns_protected", "dns_name", "forbid_reclamation", "reclaimable", "ttl", "use_ttl"}
+	return &res
+}
+
+// CreateAAAARecord takes Name, Ipv6Addr and View of the record to create AAAA Record
+// Optional fields: NetView, Ea, Cidr
+// Before creating, it checks if the Name and IP passed already exists in the network
```

Comment:
please replace it's with method names, everywhere

Reply:
Okay, I'll change that.